assets/fonts/UnifrakturMaguntia/UnifrakturMaguntia.css

```diff
@@ -0,0 +1,13 @@
+/* unifrakturmaguntia-regular - latin */
```

Comment:
the changes made to the other fonts (use only woff & woff2) need to be made to this font as well